play/tests/test_jamesk.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from play.jamesk import knockknock
+
+def test_knockknock():
+    import sys
```

Comment:
Common stdlib imports should all be at the top. For the most part imports should only be local to a function if the import is slow, and the function is not certain to run as part of the module.

Reply:
Done in 37b20472ed441bf813243e3daadb605296cf3d64